app/models/manageiq/providers/kubernetes/container_manager/event_catcher_mixin.rb

```diff
@@ -6,7 +6,8 @@ module ManageIQ::Providers::Kubernetes::ContainerManager::EventCatcherMixin
     'Node'                  => %w(NodeReady NodeNotReady Rebooted NodeSchedulable NodeNotSchedulable InvalidDiskCapacity
                                   FailedMount),
     'Pod'                   => %w(Scheduled FailedScheduling FailedValidation HostPortConflict DeadlineExceeded
-                                  FailedSync Created Failed Started Killed Stopped Unhealthy),
+                                  FailedSync OutOfDisk NodeSelectorMismatching InsufficientFreeCPU
+                                  InsufficientFreeMemoryCreated Failed Started Killed Stopped Unhealthy),
```

Comment:
@dkorn I suppose there should be a space between `InsufficientFreeMemory` and `Created`